src/components/OnboardingDialog/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import { radius, shadow, space, duration, transition } from '../../theme'
+import { useTransition } from '../../hooks'
+
+const BodyWrapper = styled.div`
+  background-color: white;
+  border-radius: ${radius.lg};
+  box-shadow: ${shadow.strong};
+  padding: ${space[32]};
+`
+
+const OnboardingDialogBody = ({ children }) => (
+  <BodyWrapper>{children}</BodyWrapper>
+)
+
+OnboardingDialogBody.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+}
+
+const DialogWrapper = styled.div`
+  position: absolute;
+  top: ${props => space[props.positionTop]};
```

Comment:
Position top is also dependant on screen-size, so maybe remove the prop, and just do fixed for mobile/desktop 🤔

Reply:
Isn't dependant on the child content? I've added it to be able to make adjustments to fit it better with underlaying content.

Reply:
Then we should make it possible to make a styled DialogWrapper. You can do this by spreading the rest props on the wrapper.

Reply:
Resolved by passing props to DialogWrapper so you can use a styled component instead of you want to alter it.